man/systemd-cryptenroll.xml

```diff
@@ -401,6 +403,31 @@
         <xi:include href="version-info.xml" xpointer="v248"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--fido2-salt-file=<replaceable>PATH</replaceable></option></term>
+
+        <listitem><para>When enrolling a FIDO2 security token, specifies the path to a file or an
+        <constant>AF_UNIX</constant> socket from which we should read the salt value to be used in the
+        HMAC operation performed by the FIDO2 security token. If this option is not specified, the salt
+        will be randomly generated.</para>
+
+        <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>--fido2-token-parameters=<replaceable>BOOL</replaceable></option></term>
```

Comment:
so, hmm, i think this gets a bit confusing, because the word "token" is used for two similar but different things:

1. fido2 tokens is what many people call fido2 security keys
2. luks2 tokens is a special data structure for encoding additional info about enrolled slots in the superblock

here the "token" in the "--fido2-token-parameters=" refers to the latter I guess, but people likely will read it as the former (at least I did so, and it took me a while to grap this).

luks2 tokens are a concept only people knowing LUKS well will know. I think we should say "superblock" here, or maybe "header" or so. luks/cryptsetup calls the initial data structure sometimes superblock, and sometimes header (probably the latter more).

hence maybe --fido2-parameters-in-header=? or --fido2-parameters-in-superblock=?

Reply:
(and i am aware i initially suggested `--save-in-token=` even. But looking at this I think that wasn't smart ;-))

Reply:
Agreed, this could lead to some confusion. Went with `--fido2-parameters-in-header=`.